<launch>

<!-- Run pass through filter node -->
<node pkg = "PCL_Perception" name = "passThrough_filter" type = "pass_through" />

<!-- Run planar segmentation node -->
<node pkg = "PCL_Perception" name = "planarSegmentation_node" type = "planar_segmentation_node" />

</launch>

